man/systemd.xml

```diff
@@ -1482,7 +1485,26 @@
   </refsect1>
 
   <refsect1>
-    <title>Sockets and FIFOs</title>
+    <title>System clock epoch</title>
+
+    <para>When <command>systemd</command> is started or restarted, it may set the system clock to the
+    "epoch". This mechanism is used to ensure that the system clock remains somewhat reasonably initialized
+    and roughly monotonic across reboots, in case no battery-backed local RTC is available or it does not
+    work correctly.</para>
+
+    <para>The epoch is the lowest date below above which the system clock time is assumed to be set
```

Comment:
"lowest date above which"

Reply:
Arrgh. I tried to reword this too many times. I'll push in a moment. I think we don't have to run CI again. It seems to be passing now.

Reply:
… and it's even marked properly. Just ignore my rambling please.